homeassistant/components/zwave/__init__.py

```diff
@@ -627,6 +628,39 @@ def value_changed(self, value):
         self.update_properties()
         self.schedule_update_ha_state()
 
+    def value_handler(self, method='get', class_id=None, index=None,
```

Comment:
I just mean something small like this, just to make it easier to read, but still keeping all the code common. Up to you, though.

```python
def get_value(**kwargs)
    return self._value_handler(method='get', **kwargs)
```